src/Symfony/Component/VarDumper/Caster/ReflectionCaster.php

```diff
@@ -293,7 +293,7 @@ public static function castParameter(\ReflectionParameter $c, array $a, Stub $st
             try {
                 $a[$prefix.'default'] = $v = $c->getDefaultValue();
                 if ($c->isDefaultValueConstant()) {
```

Comment:
shouldn't this line be changed to `if ($c->isDefaultValueConstant() && !\is_object($v)) {` instead?
can give it a try and see how that displays things?
can you also please add a test case to cover this?

Reply:
I'll try :)

Reply:
@nicolas-grekas Yes, your code is better, now I've displayed object instead of class :) Now I will try to create test case :)